Remora.Discord.Commands/Extensions/CommandTreeExtensions.cs

```diff
@@ -608,7 +608,7 @@ ILocalizationProvider localizationProvider
             var minValue = parameter.Parameter.GetCustomAttribute<MinValueAttribute>();
             var maxValue = parameter.Parameter.GetCustomAttribute<MaxValueAttribute>();
 
-            if (discordType is not Integer or Number && (minValue is not null || maxValue is not null))
+            if ((discordType is not Integer && discordType is not Number) && (minValue is not null || maxValue is not null))
```

Comment:
```suggestion
            if ((discordType is not (Integer or Number) && (minValue is not null || maxValue is not null))
```

Reply:
addressed in [bf6bc7d](https://github.com/Remora/Remora.Discord/pull/235/commits/bf6bc7d7ca4252876a7e0f3d1081ba7dcf646b8a)